target-platforms/target-platform-latest/target-platform-latest.target

```diff
@@ -30,6 +30,10 @@
 			<unit id="org.eclipse.sdk.ide" version="0.0.0"/>
 			<repository location="https://download.eclipse.org/eclipse/updates/latest/"/>
 		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<unit id="org.eclipse.tm4e.feature.feature.group" version="0.0.0"/>
+			<repository location="https://download.eclipse.org/tm4e/releases/latest/"/>
+		</location>
```

Comment:
Adding this does not make it visible to the SemanticHighlightReconcilerStrategy, but I do not know why. In any case, if we do not want the dependency on TM4E, it would not matter much, but if my approach is fine, I am happy if you know the cause of the build failure.

Reply:
I didn't see any issue locally, I had to add the necessary `import org.eclipse.tm4e.ui.ITheme` line but then everything seemed fine.